From 8eee6891d456fb03c4cbc8e0f4625ba157049f6d Mon Sep 17 00:00:00 2001
From: Pantelis Antoniou <panto@antoniou-consulting.com>
Date: Tue, 8 Jan 2013 20:40:45 +0200
Subject: [PATCH 51/98] OF: display timings helper convert to get child node

of_find_node_by_name() is not what should be used here.
It does a global name search which is not what we want to do here.

Convert to the use the correct of_get_child_by_name() function.

Signed-off-by: Pantelis Antoniou <panto@antoniou-consulting.com>
---
 drivers/of/of_display_timings.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/of/of_display_timings.c b/drivers/of/of_display_timings.c
index 388fe4c..d767272 100644
--- a/drivers/of/of_display_timings.c
+++ b/drivers/of/of_display_timings.c
@@ -108,7 +108,7 @@ struct display_timings *of_get_display_timing_list(struct device_node *np)
 		return NULL;
 	}
 
-	timings_np = of_find_node_by_name(np, "display-timings");
+	timings_np = of_get_child_by_name(np, "display-timings");
 	if (!timings_np) {
 		pr_err("%s: could not find display-timings node\n", __func__);
 		return NULL;
-- 
1.8.1

